nevergrad/optimization/experimentalvariants.py

```diff
@@ -329,3 +331,9 @@
 HCHAvgLHSSearch = SamplingSearch(sampler="LHS", recommendation_rule="average_of_hull_best").set_name("HCHAvgLHSSearch", register=True)
 HCHAvgCauchyLHSSearch = SamplingSearch(sampler="LHS", cauchy=True, recommendation_rule="average_of_hull_best").set_name(
     "HCHAvgCauchyLHSSearch", register=True)
+
+
+# Split on top of competence map.
+MetaNGOpt8 = ConfSplitOptimizer(multivariate_optimizer=NGOpt8, monovariate_optimizer=NGOpt8).set_name("MetaNGOpt8", register=True)
```

Comment:
if it's only for one opt group, can you define it there? If you plan to add it to more optgroups, then let's leave it here.

Reply:
The plan is to have this as the main tested optimization method for splitting a problem into collab. coevolution, so either adding it inside "competitive" and "baselines" or adding it directly in benchmarks/experiments.py. For the moment I add it inside "competitive" and "baselines".